### PR TITLE
[Storage] Fix VLW test on java 11.

### PR DESCRIPTION
**Pull request by kasobol-msft**

### CODE REVIEW

**Review by gapra-msft — APPROVED** [review]

File: sdk/storage/azure-storage-blob/pom.xml [hunk 1]
```diff
@@ -313,6 +313,7 @@
                 --add-opens com.azure.storage.common/com.azure.storage.common.implementation=ALL-UNNAMED
                 --add-opens com.azure.storage.common/com.azure.storage.common.sas=ALL-UNNAMED
                 --add-opens com.azure.storage.blob/com.azure.storage.blob=ALL-UNNAMED
+                --add-opens com.azure.storage.blob/com.azure.storage.blob=com.fasterxml.jackson.databind
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob/pom.xml:316]: What was causing the test to fail? Was test code inaccessible to Jackson or source classes? If it was the latter this should be an `opens to` in the `module-info`.

Comment (kasobol-msft) [reply sdk/storage/azure-storage-blob/pom.xml:316]: @alzimmermsft this line https://github.com/Azure/azure-sdk-for-java/blob/77efff64a75d8c97041e69d587d077451827033a/sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ImmutableStorageWithVersioningTest.groovy#L90

**Review by alzimmermsft — APPROVED** [review]

File: sdk/storage/azure-storage-blob/pom.xml (hunk 1 above)
Comment (kasobol-msft) [reply sdk/storage/azure-storage-blob/pom.xml:316]: It seems that https://github.com/Azure/azure-sdk-for-java/pull/23125 is also fixing the problem, so we'll try that first and use this PR as a backup.

**Comment by kasobol-msft** [comment]
We'll use this if https://github.com/Azure/azure-sdk-for-java/pull/23125 isn't effective.

**Pull request closed without merging**